experimental/GModule/GModule.jl

```diff
@@ -299,10 +263,6 @@ function invariant_lattice_classes(M::GModule{<:Oscar.GAPGroup, <:AbstractAlgebr
   return invariant_lattice_classes(MZ)
 end
 
-function Oscar.pseudo_inv(h::Generic.ModuleHomomorphism)
```

Comment:
For now, but eventually, this will have to be done differently as I'd like to have inv(inv()) to be the identity